packages/volto/src/components/manage/Blocks/Search/SearchBlockView.jsx

```diff
@@ -42,7 +42,8 @@ const blockPropsAreChanged = (prevProps, nextProps) => {
   return isEqual(prev, next);
 };
 
-const applyDefaults = (data, root) => {
+const applyDefaults = (data, root, block_query) => {
+  block_query = block_query?.length ? block_query : [];
```

Comment:
It is better to define a new variable than to modify the parameter.

```suggestion
  const normalizedBlockQuery = blockQuery?.length ? blockQuery : [];
```